src/main/scala/com/github/pjfanning/pekkobuild/PekkoDevelocityPlugin.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.pjfanning.pekkobuild
+
+import com.gradle.develocity.agent.sbt.DevelocityPlugin
+import com.gradle.develocity.agent.sbt.DevelocityPlugin.autoImport.{
+  develocityConfiguration,
+  FlakyTestPolicy,
+  ProjectId,
+  Publishing
+}
+import sbt.{AutoPlugin, Plugins, PluginTrigger, Setting}
+import sbt.Keys.insideCI
+
+import java.net.URI
+
+object PekkoDevelocityPlugin extends AutoPlugin {
+
+  private val ApacheDevelocityUrl   = URI.create("https://ge.apache.org").toURL
+  private val PekkoProjectId        = ProjectId("pekko")
+  private val ObfuscatedIPv4Address = "0.0.0.0"
+
+  override lazy val trigger: PluginTrigger = allRequirements
+  override lazy val requires: Plugins      = DevelocityPlugin
+
+  override lazy val buildSettings: Seq[Setting[_]] = Seq(
+    develocityConfiguration := {
+      val isCI = insideCI.value
+
+      val original = develocityConfiguration.value
+      val apacheDevelocityConfiguration =
+        original
+          .withProjectId(PekkoProjectId)
+          .withServer(
+            original.server
+              .withUrl(Some(ApacheDevelocityUrl))
+              .withAllowUntrusted(false)
+          )
+          .withBuildScan(
+            original.buildScan
+              .withPublishing(Publishing.onlyIf(_.authenticated))
+              .withBackgroundUpload(!isCI)
+              .withObfuscation(
+                original.buildScan.obfuscation
+                  .withIpAddresses(_.map(_ => ObfuscatedIPv4Address))
+              )
+          )
+      if (isCI) {
```

Comment:
I'd include a comment that that helps making it explicit that test retry (and flaky test detection in general) is enabled only on the CI (and hence will be reported only on build scans produced from the CI).

Reply:
Do you mean the comment in the code here, or in the readme?

Reply:
I meant to inline a comment here in the build logic.